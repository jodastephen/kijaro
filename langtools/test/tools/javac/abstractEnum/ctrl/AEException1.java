/*
 * Copyright (c) 2007 Frederic Simon.  All Rights Reserved.
 * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
 *
 * This code is free software; you can redistribute it and/or modify it
 * under the terms of the GNU General Public License version 2 only, as
 * published by the Free Software Foundation.
 *
 * This code is distributed in the hope that it will be useful, but WITHOUT
 * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
 * version 2 for more details (a copy is included in the LICENSE file that
 * accompanied this code).
 *
 * You should have received a copy of the GNU General Public License version
 * 2 along with this work; if not, write to the Free Software Foundation,
 * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
 *
 * Please contact Sun Microsystems, Inc., 4150 Network Circle, Santa Clara,
 * CA 95054 USA or visit www.sun.com if you need additional information or
 * have any questions.
 *
 */
package abstractEnum.ctrl;

/*
 * @test @(#)AEException1.java 1.7 08/06/07
 * @bug 6507006
 * @summary abstract enum should not support members
 * @author freds
 *
 * @compile/fail/ref=AEException1.out -XDstdout -XDdiags=%b:%l:%_%m AEException1.java
 */
public abstract enum AEException1 {
    one, two;
}