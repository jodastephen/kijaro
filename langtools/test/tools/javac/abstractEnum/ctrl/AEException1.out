AEException1.java:35: <identifier> expected
    one, two;
       ^
1 error
